eth/protocols/eth/protocol.go

```diff
@@ -39,11 +42,11 @@ const ProtocolName = "eth"
 
 // ProtocolVersions are the supported versions of the `eth` protocol (first
 // is primary).
-var ProtocolVersions = []uint{ETH66}
+var ProtocolVersions = []uint{ETH100, ETH68, ETH67, ETH66}
```

Comment:
This should be ETH100, ETH66 only. We will transition from ETH66 to ETH100 directly.